src/exercises/day-3/safe-ffi-wrapper.rs

```diff
@@ -14,6 +14,9 @@
 
 // ANCHOR: solution
 // ANCHOR: ffi
+
+use tempfile::tempfile;  // requires tempfile = "3" in Cargo.toml dependencies
```

Comment:
We test the solution using the [`Cargo.toml` in `src/exercises`](https://github.com/google/comprehensive-rust/blob/main/src/exercises/Cargo.toml) and this line is not actually needed — the test code below uses the `tempfile` crate directly.

If you would want to import `tempfile::TempDir` for the tests, then you should add the `use tempfile::TempDir` line _inside_ `mod tests`. This way it becomes possible to distinguish between regular dependencies and dev-dependencies. The latter are only used by `cargo test` and won't burden other users of the code.

T